### DISCUSSION TITLE
Option to disable email notification about issue converted into discussion

### DISCUSSION THREAD

**Discussion by CyrusYip** [community/community#114510] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

Some people provide a comment service for websites via GitHub Issues (example: https://github.com/utterance/utterances). I comment a lot on those sites. When the a repo owner convert many issues into discussions, I will receive so many email notifications telling me "`@someone` converted this issue into discussion `#2`". This is quite bothering. I don't need to know such conversion and only care about if there is new comment.

Can we have the option for repo owners to decide whether to send this conversion email notification? The notification is useless in this comment service scenario.

**Comment by 0YJ** [comment, marked as answer]
You can set up a rule to automatically archive or delete emails containing specific phrases such as "converted this issue into a discussion" or go to your GitHub profile, access your settings, navigate to the Notifications tab, and adjust your email preferences.
